setup.cfg

```diff
@@ -46,7 +46,7 @@ install_requires =
     nbformat
     packaging
     psutil
-    pulp >=2.0
+    pulp >=2.3.1,<2.9 # pulp introduced a breaking change from 2.7 to 2.8. Hence we should always pin to the minor version.
```

Comment:
Mirrored to bioconda here: https://github.com/bioconda/bioconda-recipes/pull/45234

Unfortunately not already for 8.1.2, just for 8.1.3